lib/v2/neu/radar.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'www.bmie.neu.edu.cn': {
+        _name: '东北大学医学与生物信息工程学院',
+        '.': [
+            {
+                title: '学院新闻',
```

Comment:
```suggestion
                title: '学院新闻 - 医学与生物信息工程学院',
```
`title` should match the H3 header